screens/ContentTypesScreen/ContentTypesScreen.js

```diff
@@ -149,16 +146,18 @@ const ContentTypesScreen = (props) => {
 
     const returnListData = useCallback(() => {
         const dataIsNotEmpty = data && data.length > 0;
+        console.log('data', data);
+        console.log(dataIsNotEmpty);
```

Comment:
```suggestion
```
clog

Reply:
Thanks for pointing that out. Sorry for that happy accident 😅